generators/generator-bot-empty/README.md

```diff
@@ -1,11 +1,37 @@
 # @microsoft/generator-bot-empty [![NPM version][npm-image]][npm-url]
-> Yeoman generator for creating an Adaptive bot built on the Azure Bot Framework using the Empty template.
 
-This template instantiates an empty bot with no dependent packages.
+> Yeoman generator for creating an empty bot built on the Azure Bot Framework & Composer.
```

Comment:
I was a bit worried about this too, but since we're not going to list adaptive in Composer, I don't think it will be an issue. Anyone who's exploring doing this from the command line and/or creating their own will hopefully read further.